src/routeApp/routeOptions.js

```diff
@@ -0,0 +1,39 @@
+const RouteOption = [
```

Comment:
Can you elaborate on the use of this file and the information. If its for testing, it should be in a `test` folder instead of being at the route folder of your app